release_notes/ocp-4-16-release-notes.adoc

```diff
@@ -119,6 +119,46 @@ With this release, migration from the OpenShift SDN network plugin to OVN-Kubern
+
+For more information, see xref:..//storage/container_storage_interface/persistent-storage-csi-vsphere.adoc#vsphere-change-max-snapshot_persistent-storage-csi-vsphere[Changing the maximum number of snapshots for vSphere]
+
+[id="ocp-4-16-pv-last-phase-transition-time"]
+==== Persistent volume last phase transition time parameter (Technology Preview)
+In {product-title} 4.16 introduces a new parameter `LastPhaseTransitionTime`, which has a timestamp that is updated every time a persistent volume (PV) transitions to a different phase (`pv.Status.Phase`). This feature is being released with Technology Preview status.
+
+[id="ocp-4-16-storage-csi-smb-cifs-driver-operator"]
+==== Persistent storage using SMB CIFS CSI Driver Operator (Technology Preview)
+{Product-title} is capable of provisioning persistent volumes (PVs) using the Container Storage Interface (CSI) driver for the Server Message Block (SMB) protocol Common Internet File System (CIFS) dialect. The SMB CIFS CSI Driver Operator that manages this driver is in Technology Preview.
+
+// For more information, see xref:..//storage/container_storage_interface/persistent-storage-csi-smb-cifs.adoc#persistent-storage-csi-smb-cifs[SMB CIFS CSI Driver Operator].
+
+[id="ocp-4-16-storage-rwop-selinux-context-mode"]
+==== RWOP with SELinux context mount is generally available
+{product-title} 4.14 introduced a new access mode with Technical Preview status for persistent volumes (PVs) and persistent volume claims (PVCs) called ReadWriteOncePod (RWOP). RWOP can be used only in a single pod on a single node compared to the existing ReadWriteOnce access mode where a PV or PVC can be used on a single node by many pods. RWOP uses SELinux context mount, which allows the driver to mount the volume directly with the correct SELinux labels, eliminating the need to recursively relabel the volume, and pod startup can be significantly faster. 
+
+In {product-title} 4.16, this feature is generally available.
+
+For more information, see xref:..//storage/understanding-persistent-storage.adoc#pv-access-modes_understanding-persistent-storage[Access modes].
+
+[id="ocp-4-16-storage-csi-vsphere-compliance-check"]
+==== vSphere 3.1 Updated CSI topology requirements
+To support VMware vSphere Container Storage Interface (CSI) volume provisioning and usage in multi-zonal clusters, the deployment should match certain requirements imposed by CSI driver. These requirements have changed in 3.1.0, and while {product-title} 4.16 accepts both the old and new tagging methods, you should use the new tagging method since VMware considers the old way an invalid configuration, and therefore to prevent problems you should not use it.
```

Comment:
```suggestion
To support VMware vSphere Container Storage Interface (CSI) volume provisioning and usage in multi-zonal clusters, the deployment should match certain requirements imposed by CSI driver. These requirements have changed from 3.1.0, and while {product-title} 4.16 accepts both the old and new tagging methods, you should use the new tagging method since VMware considers the old way an invalid configuration, and therefore to prevent problems you should not use it.
```

My only concern is in OCP4.16, we used vSphere CSI Driver 3.1.2. The current wording might inadvertently lead users to believe that we are using version 3.1.0. Please feel free to make any necessary updates to avoid any potential misunderstanding.

Reply:
I would consider breaking apart this last sentence also.